.github/workflows/main.yml

```diff
@@ -15,3 +44,4 @@ jobs:
           columns: 3
           object-field-names: '[ "githubUsername", "name", "imageUrl", "issueNumber" ]'
           html-cell: '<td align="center"><p><a href="https://github.com/{{ githubUsername }}">{{ name }}</a></p><img src="{{ imageUrl }}" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/{{ issueNumber }}">(💯 give your vote)</a></p></td>'
+
```

Comment:
There's an unneeded extra line here (the Git diff doesn't display the endline, meaning there are two lines).
```suggestion
```